### PR TITLE
Thresholding Tutorial using inRange function on a video

### PR DESCRIPTION
**Pull request by rishirajsurti**
New tutorial for ImgProc module which involves Threshold using inRange function. This can also be used for object detection.

### CODE REVIEW

**Comment by mshabunin** [comment]
Please, use [snippet](http://docs.opencv.org/master/d4/db1/tutorial_documentation.html#tutorial_documentation_commands_include) doxygen command to include code from .cpp file.

**Comment by rishirajsurti** [comment]
Thank you for the feedback. I've made the necessary changes.

File: samples/cpp/tutorial_code/ImgProc/Threshold_inRange.cpp [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#include "opencv2/imgproc/imgproc.hpp"
+#include "opencv2/highgui/highgui.hpp"
+
```
Comment (sturkmen72) [inline samples/cpp/tutorial_code/ImgProc/Threshold_inRange.cpp]: using OpenCV 3 headers will be better
```
#include "opencv2/imgproc.hpp"
#include "opencv2/highgui.hpp"
```

Comment (rishirajsurti) [inline samples/cpp/tutorial_code/ImgProc/Threshold_inRange.cpp]: @sturkmen72 Yes. You are correct. Thanks for pointing out.

**Comment by mshabunin** [comment]
Please, squash all commits into one.

**Comment by rishirajsurti** [comment]
@mshabunin Done!

**Comment by mshabunin** [comment]
:+1:

**Comment by rishirajsurti** [comment]
Thanks! This is my first contribution to OpenCV!

**Pull request merged**